business_requirement_deliverable/__openerp__.py

```diff
@@ -6,7 +6,7 @@
     'category': 'Business Requirements Management',
     'summary': 'Manage the Business Requirement Deliverables and \
                 Resources for your customers',
-    'version': '8.0.4.0.0',
+    'version': '8.0.4.0.1',
```

Comment:
I think we should bump to 8.0.5.0.0 because this modules implies a manual migration of the data